core/quantity/src/Parser.ts

```diff
@@ -400,17 +472,17 @@ export class Parser {
     if (tokens.length === 1) {
       if (tokens[0].isNumber) {

```

Comment:
nit: mind cleaning up the consecutive if statements here, makes it hard to parse through the code on review. Too many braces :)

Reply:
They can't be combined in this case, because each if has separate else cases.

Reply:
ah okay, missed that.